### PR TITLE
Assignment 1 - ckappes

### PR DESCRIPTION
**Pull request by CKappes-Git**

### CODE REVIEW

File: wordle-clone/src/views/HomeView.vue
Comment (Hex-Paladin) [inline wordle-clone/src/views/HomeView.vue:1]: I think you are supposed to use a v-card to display the about section. Also you could create a router link here to directly connect the user to the game with a larger button, Overall definitely looking good!

**Comment by DominicAlvarez** [comment]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Jest ✅
Create a simple page called game.vue ❌✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ❌✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅ 

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅ 

Pipeline runs on each check in
Jest tests run and pass ❌✅ ?
Vue app builds ❌✅ ?
Vue app deploys to Azure ❌✅ ?
Azure site works ❌✅ ?

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ❌

Looking through the files of your work it seems like everything is there and that there is a ci/cd pipeline. However no link to the website on the pull request nor the readme of the assignment1 branch.  Im assuming that it does build and is deployed but need the link to prove that. Everything else looks fine just need to add the link on readme.

**Review by czamora7 — COMMENTED** [review]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅

Nice work!

File: wordle-clone/src/views/HomeView.vue
Comment (CKappes-Git) [reply wordle-clone/src/views/HomeView.vue:1]: I don't see anything in the assignment about changing the about page, but I put it all in a v-card under the assumption that I have misunderstood or forgotten something.  I did also bump up the size of the router links though and I do think that looks a bit nicer.

File: wordle-clone/src/game.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
```
Comment (mmwoodfo) [inline wordle-clone/src/game.vue:1]: This should be under your views folder and the file name should be PascalCase `Game.vue`

File: wordle-clone/src/game.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import { RouterLink, RouterView } from 'vue-router'
```
Comment (mmwoodfo) [inline wordle-clone/src/game.vue:2]: You shouldn't need to import this!

File: wordle-clone/src/game.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+</script>
+
+<template>
+
```
Comment (mmwoodfo) [inline wordle-clone/src/game.vue:6]:
```suggestion
```

File: wordle-clone/src/game.vue [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        <v-card-item>
+            <div class="text-caption"> I mean, it's probably harder in a lot of respects, but I've never lost a game of wordle while I have lost games of hangman.  Still, I'm not sure if that says more about me or the game. </div>
+        </v-card-item>
+        <v-card-actions>
+            <v-btn variant="outlined" to="/">
+                Home
+            </v-btn>
+        </v-card-actions>
+    </v-card>
+
```
Comment (mmwoodfo) [inline wordle-clone/src/game.vue:20]:
```suggestion
```

File: wordle-clone/src/game.vue [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+            <v-btn variant="outlined" to="/">
+                Home
+            </v-btn>
+        </v-card-actions>
+    </v-card>
+  
+</template>
+
+
+<style scoped>
```
Comment (mmwoodfo) [inline wordle-clone/src/game.vue:24]: Do you need all this styling?

**Pull request closed without merging**